app/lucy/src/app/components/Routes/admin-tools/admin-tools.component.html

```diff
@@ -1,90 +1,76 @@
-      </mat-toolbar-row>
-    </mat-toolbar>
-  </div>
-</div>
-<br />
-<div class="container">
-  <!-- Data managment-->
-  <div class="row box">
-    <div class="col-12 section-title-text">
-      Manage Data
-    </div>
-    <div class="col-12">
-      <hr style="width: 100%; color: black; height: 1px; background-color:black;" />
-    </div>
-    <div class="col-12 col-sm-6 col-lg-10 section-body-text">
-      There are currently {{numberOfDataInInspectAppToExport}} records in the Inspect App Database.
-    </div>
-    <div class="col-12 col-sm-6 col-lg-2">
-      <button mat-raised-button class="action-button" color="primary" (click)="export()"> Export All Data </button>
+<div class="admin-container">
+  <!-- Header Section -->
+  <div class="container-fluid header-container sticky animated">
+    <div class="container">
+      <mat-toolbar class="toolbar">
+        <mat-toolbar-row>
+          <span class="animated fadeIn">Admin Tools</span>
+        </mat-toolbar-row>
+      </mat-toolbar>
     </div>
   </div>
-  <!-- End Data managment -->
   <br />
-  <!-- Requests -->
-  <div class="row box">
-    <div class="col-12 section-title-text">
-      New Users
-    </div>
-    <div class="col-12">
-      <hr style="width: 100%; color: black; height: 1px; background-color:black;" />
-    </div>
-    <div class="col-12">
-      <table class="table table-borderless table-hover">
-        <thead>
-          <tr>
-            <th scope="col">Username</th>
-            <th scope="col">Name</th>
-            <th scope="col">Current Role</th>
-            <th scope="col">Requested Role</th>
-            <th scope="col">Reasons</th>
-            <th scope="col"></th>
-          </tr>
-        </thead>
-        <tbody>
-            <tr app-request-cell *ngFor="let request of requests; let i = index" [request]="request" (shouldRefresh)="requestResponseSent()">
-        </tbody>
-      </table>
-    </div>
-  </div>
-  <!-- End Requests -->
-  <br />
-  <!-- All Users -->
-  <div class="row box">
-    <div class="col-12 section-title-text">
-      All Users
-    </div>
-    <div class="col-12">
-      <hr style="width: 100%; color: black; height: 1px; background-color:black;" />
-    </div>
-    <div class="col-12">
-      <table class="table table-borderless table-hover">
-        <thead>
-          <tr>
-            <th scope="col">Username</th>
-            <th scope="col">First Name</th>
-            <th scope="col">Last name</th>
-            <th scope="col">Email</th>
-            <th scope="col">Role</th>
-            <th scope="col"></th>
-          </tr>
-        </thead>
-        <tbody>
-          <tr app-user-cell *ngFor="let user of allUsers; let i = index" [user]="user">
-          </tr> 
-        </tbody>
-      </table>
+
+  <div class="container">
+    <div class="row">
+      <div class="col-3 side-menu animated">
+        <div>
+          <p class="text text-header">INVASIVES BC</p>
```

Comment:
Confirmed with Roop - this text should be written as "Invasives BC" and then styled in CSS to be title case so that screen readers don't yell out the title